Dockerfile

```diff
@@ -42,6 +42,14 @@ COPY --from=provider-ui /out /app/provider-ui/out
 COPY --from=wrk2 /wrk2 /app/cmd/wrk2
 COPY --from=wrk2 /wrk2/wrk /usr/local/bin
 COPY --from=nighthawk /nighthawk-go/apinighthawk/bin /usr/local/bin
+RUN curl -Lo nighthawk_service-ubuntu-amd64-v1.0.1.tar.gz https://github.com/layer5io/getnighthawk/releases/download/v1.0.1/nighthawk_service-ubuntu-amd64-v1.0.1.tar.gz \

```

Comment:
Pinning to a specific version seems like something that we don't want to do. Will you give this additional thought and circle back with an alternate strategy?

Reply:
sure